src/ol/controls/Copyright.js

```diff
@@ -17,7 +17,10 @@ class Copyright extends mixin(CommonControl) {
 
   activate() {
     super.activate();
-    this.addCopyrightContainer(this.map.getTargetElement());
```

Comment:
We should assume the target is always there. With this condition the control fails silently if that's not the case. Please try fixing it in the test instead.